engine/xml-config/src/main/resources/configuration.xsd

```diff
@@ -140,6 +171,27 @@
 		</sequence>
 	</complexType>
 
+    <complexType name="remoteComponentsType">
+        <sequence>
+            <element name="server" type="ab:serverType"/>
+        </sequence>
+    </complexType>
+
+    <complexType name="serverType">
+        <sequence>
+            <element name="url" type="string" minOccurs="0" maxOccurs="1">
+                <annotation>
+                    <documentation>
+                        Defines URL to the Datacleaner monitor server
```

Comment:
Actually I am already wondering if this is the base URL (such as http://localhost:8080/DataCleaner-monitor) or the URL to a specific web service or so. Maybe we should add that to this description.

And very minor: The 'c' in DataCleaner should be capitalized.